cmd/zpool/zpool_main.c

```diff
@@ -6470,6 +6481,13 @@ status_callback(zpool_handle_t *zhp, void *data)
 			    "run 'zpool scrub'.\n"));
 			break;
 
+		case ZPOOL_ERRATA_ZOL_6845_ENCRYPTION:
+			(void) printf(gettext("action: To correct the issue "
+			    "revert to an earlier version and backup\n\tand "
+			    "destroy any existing encrypted datasets before "
```

Comment:
I think `existing` is redundant (same in show_import()).

Reply:
Ditto, see show_import comment.